.github/workflows/base.yml

```
@@ -28,42 +28,96 @@ jobs:
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Docker metadata
+        id: metadata
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            winglian/axolotl-base
+            axolotlai/axolotl-base
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)